CONTRIBUTING.md

```diff
@@ -38,10 +38,10 @@ This project and everyone participating in it is governed by the [Code of Conduc
 
 We have an official tutorial and a detailed FAQ compiled from the community. 
 
-* [MACS documentations](https://macs3.github.io)
-  * [MACS tutorial](https://macs3.github.io)
-  * [MACS FAQ](https://macs3.github.io)
-  * [MACS API](https://macs3.github.io)
+* [MACS documentations](https://macs3-project.github.io/MACS/)
+  * [MACS tutorial](https://macs3.github.io) <!-- change link? -->
+  * [MACS FAQ](https://macs3.github.io) <!-- change link? -->
+  * [MACS API](https://macs3.github.io) <!-- change link? -->
```

Comment:
Currently, we don't have that. We may need to use Sphinx tool to generate documents for the API and link to it.